lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -391,7 +391,7 @@ class CanvasKitRenderer implements Renderer {
     final Rasterizer rasterizer =
         _getRasterizerForView(view as EngineFlutterView);
 
-    rasterizer.draw((scene as LayerScene).layerTree);
+    rasterizer.draw((scene as LayerScene).layerTree, size: size);
```

Comment:
@harryterkelsen is there a way of testing this? Somehow asserting that the thing that was drawn matches the size of the incoming `size`?

Reply:
IIRC the size is only used to size the output canvases before drawing to them, so if you wanted to test this then I think you should call `draw` with an explicit size and then check that the bitmap created and the bitmaprenderer canvases are the same size as the `size` that is passed in.